README.md

```diff
@@ -0,0 +1,23 @@
+# NI JavaScript Style Guide
+
+Welcome to NI's internal and external JavaScript and TypeScript linter rules for [ESLint](https://eslint.org/docs/user-guide/getting-started) and [TSLint](https://palantir.github.io/tslint/).
+
+## Installation
+
+```
+npm install --save-dev eslint eslint-plugin-import @ni/eslint-config
+```
+
+## Usage
+
+Extend @ni in the ESLint configuration.
+
+```json
+{
+    "extends": "@ni"
+}
+```
+
```

Comment:
At some point (doesn't have to be in this initial submission) we should come up with contributing guidelines. Some ideas of what they'd contain:
 - how do I propose a new rule? who approves it?
 - how do we version this repo? What constitutes a major/minor/patch change?
 - what's our overarching philosophy? why do we want standard style across NI? how do we make decisions about which rules should be active?

I don't necessarily have answers to all of those, just putting it on your radar.

Reply:
Absolutely, I kept this PR small, functional but basic. Those are things we can tackle next.

Reply:
@alejandro5042 will do this